recipes/cmcrameri/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cmcrameri" %}
+{% set version = "0.12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9c7a7533647629ec620128c4adbeb03ffc308e4ffef7dab2af7fb87176977d65
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<36]
+
+requirements:
+  host:
+    - matplotlib
+    - numpy
+    - python
+    - pip
+  run:
+    - matplotlib
+    - numpy
+    - python
+
+test:
+  imports:
+    - cmcrameri
+
+about:
+  home: https://github.com/callumrollo/cmcrameri
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Perceptually uniform colourmaps for geosciences'
+
```

Comment:
Sorry for being nit-picky, but since you will be modifying the recipe, please remove the empty lines within section (e.g. after `summary`), and add new line between sections, e.g. before `extra` below.